ClinGen/clincoded
# Connects to #1496. Issues addressed in R13 release candidate.

**Technical notes:**
1. Removed the logic to display 2 different versions of the yellow warning notes on the **editable classification matrix page** given the presence of a saved classification. The expected result is there is only one version of the yellow warning note on the **editable classification matrix page** _with or without_ a saved classification.
2. Added new curators to the test users data.
3. The GCI and VCI help docs have been updated.

**Steps to test:**
1. Create a GDM and add 1 or 2 PMIDs.
2. Add various types of evidence and score them.
3. Click on the "View Classification Matrix" button at the upper right hand corner and expect to be taken to the Classification Matrix page with the **editable** form with 4 input elements.
4. Above the form, expect to see a Bootstrap-style yellow alert note such as below:
![image](https://user-images.githubusercontent.com/6956310/31560353-cb87e1ba-b008-11e7-8931-9f9589dda065.png)
5. Fill out the input elements as needed and save the form. Then expect to be taken to the **non-editable** Classification Matrix page.
6. Click on the **Edit Classification** button at the bottom of the page. Then expect to be taken back to the **editable** Classification Matrix page.
7. Then expect to see the same yellow alert note with the same text on the page.
8. Now go to ^/users and enter `Seigert` and/or `Charlap` in the filtered input field at the upper right hand corner.
9. Then expect to see only one entry being returned for the searched user.
